app/assets/css/_environment-header.scss

```diff
@@ -1,57 +1,32 @@
 .environment-header {
+    @extend .floating-panel;
```

Comment:
A placeholder here would be better. @extending classes can lead to larger then required style sheets.

http://thesassway.com/intermediate/understanding-placeholder-selectors